core/store/src/db.rs

```diff
@@ -811,6 +838,7 @@ pub enum StatsValue {
     Count(i64),
     Sum(i64),
     Percentile(u32, f64),
+    ColumnValue(DBCol, i64),
```

Comment:
Maybe it would make sense to change this to `u64` and move the cast to where the metric is set? I’m generally a fan of keeping types unchanged for as long as possible.